doc/build-osx.md

```diff
@@ -202,6 +202,12 @@ If `sqlite` or `qt` are not installed, this will throw an error.
 cmake -B build -DBUILD_GUI=ON
 ```
 
+### **If cmake is not detecting brew installed dependencies - try relinking them.**
```

Comment:
Is this issue CMake specific?

Reply:
good point - i will move it to the end of the "brew install" list and make a more general instruction.

-thanks